test/unit/facts.test.js

```diff
@@ -764,6 +765,54 @@ tap.test('display_host', { timeout: 20000 }, (t) => {
   })
 })
 
+tap.test('log metrics for common logging libraries', (t) => {
+  t.autoend()
+
+  const packages = ['winston', 'bunyan']
+  let getJSON
+  let agent = null
+
+  t.beforeEach(() => {
+    loggerMock.debug.reset()
+    const config = {
+      app_name: [...APP_NAMES]
+    }
+    agent = helper.loadMockedAgent(Object.assign(config, DISABLE_ALL_DETECTIONS))
+    // fake getJSON for testing
+    getJSON = agent.environment.getJSON
+    agent.environment.getJSON = (cb) => {
+      cb(null, [
+        [
+          'Packages',
+          packages.map((lib) => {
+            return `["${lib}","0.0.0"]`
+          })
+        ]
+      ])
+    }
+  })
+
+  t.afterEach(() => {
+    agent.environment.getJSON = getJSON
+    helper.unloadAgent(agent)
+  })
+
+  t.test('logs metrics for common logging libraries', (t) => {
```

Comment:
might be good to verify that package names that aren't log libraries do not increment this metric

Reply:
Ok done